src/loaders/OmeZarrLoader.ts

```diff
@@ -515,6 +529,9 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
     this.beginPrefetch(keys, level);
 
     Promise.all(channelPromises).then(() => {
+      if (this.syncChannels) {
+        onData(resultChannelIndices, resultChannelData);
+      }
```

Comment:
Left a similar comment above. If `syncChannels` is turned off, is the fetched data discarded? (Would a mismatch between the UI and loaded data occur, since pausing would stop at the currently rendered frame?)